source/mir/ndslice/selection.d

```diff
@@ -1711,11 +1711,15 @@ template IotaSlice(size_t N)
     alias IotaSlice = Slice!(N, IotaMap!());
 }
 
-// undocumented
-// zero cost variant of `std.range.iota`
+/++
+Zero cost variant of `std.range.iota`
```

Comment:
Zero cost variant of `std.range.iota` - this should be with `//` . Just add that this range is used for iotaSlice.